components/Work.js

```diff
@@ -53,11 +53,12 @@ const Work = ({ work, width = '100vw' }) => {
     ''
   ).startsWith('image/');
   const workImage = flattenImageData(get(work, 'fields.asset', {}));
-  const hoveredStateTheme = get(
+  const hoveredStateTheme = get(work, 'fields.hoveredStateTheme', 'dark');
+  const disableOverlayOnDarkTheme = get(
     work,
-    'fields.hoveredStateTheme',
-    'Dark'
-  ).toLowerCase();
+    'fields.disableOverlayOnDarkTheme',
```

Comment:
not directly related to this PR but changed these lines to not to use 'toLowerCase' unnecessarily!